File: answers/VrishtiSharma/Arrays/Q1.py
nums = [int(i) for i in input("Enter the value of integers : ").split()]
target = int(input("Enter the value of target : "))
l = []
for i in range(len(nums)):
    for j in range(i+1,len(nums)):
        if nums[i] + nums[j] == target:
            l.append(i)
            l.append(j)
            break
if len(l) == 0:
    print("Try Again!")
print(l)

File: answers/MK81201/Arrays/Q1.cpp
vector<int>nums={2,7,11,15};
   int target=9;

    void solve()
    {
        int sum=0;
        for(int i=0;i<nums.size;i++)
        {
            for(int j=i+1;j<nums.size();j++)
            {
                sum=nums[i]+nums[j];
                
                if(sum==target)
                {
                    cout<<i<<" ";
                    cout<<j;
                    return;
                }
            }
        }
    }